converters/fromZigbee.js

```diff
@@ -3455,6 +3455,29 @@ const converters = {
             return result;
         },
     },
+    sinope_GFCi_status: {
+        // TH1300ZB specific
+        cluster: 'manuSpecificSinope',
+        type: ['attributeReport', 'readResponse'],
+        convert: (model, msg, publish, options, meta) => {
+            const lookup = {0: 'off', 1: 'on'};
+            if (msg.data.hasOwnProperty('GFCiStatus')) {
+                return {GFCi_status: lookup[msg.data['GFCiStatus']]};
```

Comment:
Please change this to lower case (`gfci_status`)

Reply:
Done.

Reply:
No problem for the doc.
Can-you point me to a similar documentation page for a device with manufacturer specific attributes supported ?

Reply:
Yes, something like this: https://github.com/Koenkk/zigbee2mqtt.io/blob/master/docs/devices/DJT11LM.md#sensitivity